src/plugins/intel_gna/src/gna_device.hpp

```diff
@@ -52,8 +52,8 @@ class GNADeviceHelper : public GNAPluginNS::GNADevice {
 
     static const uint32_t TotalGna2InstrumentationPoints = 2;
     Gna2InstrumentationPoint gna2InstrumentationPoints[TotalGna2InstrumentationPoints] = {
-        Gna2InstrumentationPointHwTotalCycles,
-        Gna2InstrumentationPointHwStallCycles };
+        Gna2InstrumentationPointHwTotal,
+        Gna2InstrumentationPointHwStall};
```

Comment:
not needed as we do not exchange the library

Reply:
ok